.dockerignore

```diff
@@ -1 +1,18 @@
-/dev
+# ignore everything
+*
+
+# include rubyfiles
+!lib
+!spec
+# include gemfiles
+!Gemfile
+!imap-backup.gemspec
+!Rakefile
+
+# Include essentials
+!CHANGELOG.md
+!LICENSE
+!README.md
+
+# bin stubs
+!bin
```

Comment:
The files under `bin/stubs` may be specific to the versions in the current Gemfile.lock of the developer. I think we can exclude them for the same reasons we exclude Gemfile.lock.